### ISSUE TITLE
Nan for classification loss in training stage.

### ISSUE THREAD

**Issue by wangyuxin87** [yinghdb/EmbedMask#4]
I got NAN in training stage for classification loss, though it output a small value (0.7) in the previous steps. I find it works when I calculate the arverage value in focal loss instead of using summation. However this will provide a small classification loss (0.0001). And when I multiply it by 1000 ( get 1.5 usually), the network will output NAN again.
I have already checked my datasets (not COCO) and learning rate (2.5e-3).  Is there something wrong in my implementation?

**Comment by yinghdb** [comment]
You can try to reduce the learning rate to avoid the NAN output. If it doesn't work, you may need to adjust the loss weights of embedding-related losses, which are "MODEL.EMBED_MASK.LOSS_MASK_ALPHA" and "MODEL.EMBED_MASK.LOSS_SMOOTH_ALPHA" in the config. For example, you can test if it still outputs NAN when these loss weights are set as 0. If the NAN outputs disappear, you can test again with raising the loss weights. I hope it works for you.
